R/create.R

```diff
@@ -409,6 +424,18 @@ create_stan_data <- function(data, seeding_time, rt, gp, obs, backcalc,
   imputed_times <- cases[!(accumulate), lookup]
   accumulate <- cases$accumulate
   confirmed_cases <- cases[1:(.N - forecast$horizon)]$confirm
+  if (is.null(rt)) {
```

Comment:
This should already be tested if we are checking both the Rt and deconvolution methods?

Reply:
Yes just not explicitly that it works with lots of NAs (i.e. the current failure state).